subsys/bluetooth/host/Kconfig.gatt

```diff
@@ -142,6 +143,14 @@ config BT_GATT_READ_MULTIPLE
 	  This option enables support for the GATT Read Multiple Characteristic
 	  Values procedure.
 
+config BT_GATT_READ_MULTIPLE_VARIABLE_LENGTH
```

Comment:
Could we try to make this just a little bit shorter? E.g. `BT_GATT_READ_MULTI_VAR_LEN`?

Reply:
Yeah, its a bit long.